tools/generateMetadata.ps1

```diff
@@ -16,4 +16,4 @@ Catch
 Write-Host ""
 Write-Host "2) AutoRest information"

```

Comment:
Why is there a change to this file?

Reply:
generate.cmd uses the latest autorest version by default. And currently `autorest --list-installed` has output like this:
```
core       @microsoft.azure/autorest-core                 1.9.3          <Path>
core       @microsoft.azure/autorest-core                 2.0.4143       <Path>
extension  @microsoft.azure/autorest.azureresourceschema  1.9.5          <Path>
extension  @microsoft.azure/autorest.csharp               1.9.3          <Path>
```

Instead of the output previous versions had:
```
AutoRest Code Generator
(c) 2017 Microsoft Corporation. https://aka.ms/autorest
  Build Information
  Autorest Bootstrapper :  1.2.2
  NetCore framework :      1.0.5
  Latest Core Installed :  1.2.2
  Requested Core Version : <none>
Installed versions of AutoRest :

  1.2.2
```

So this line:
```
Write-Host "Latest version:   " (autorest --list-installed | Where {$_ -like "*Latest Core Installed*"}).Split()[-1]
```
causes failure as `Where` part returns empty set of result. This failure is written to metadata in case there is no change in tools/generateMetadata.ps1